integration-test/628-standardize-water-kinds.py

```diff
@@ -23,3 +23,33 @@ def test_reservoir(self):
         self.assert_has_feature(
             16, 10481, 25324, 'water',
             {'kind': 'lake', 'reservoir': True})
+
+    # This test is for https://github.com/tilezen/vector-datasource/pull/2047
+    def test_water_reservoir(self):
+        # Francisco Reservoir, SF
+        self.generate_fixtures(dsl.way(386662458, wkt_loads('POLYGON ((-122.421611012467 37.80395991667648, -122.420227786592 37.80413210674329, -122.420159873957 37.80408277780808, -122.420117832801 37.80383748120349, -122.42154480663 37.80365748294528, -122.421611012467 37.80395991667648))'), {
+                               u'way_area': u'6936.45', u'source': u'openstreetmap.org', u'water': u'reservoir', u'name': u'Francisco Reservoir', u'covered': u'no'}))
+
+        self.assert_has_feature(
+            16, 10481, 25324, 'water',
+            {'kind': 'lake', 'reservoir': True, 'kind_detail': 'lake'})
+
+    # This test is for https://github.com/tilezen/vector-datasource/pull/2047
+    def test_water_lagoon(self):
+        # Francisco Reservoir, SF
+        self.generate_fixtures(dsl.way(386662458, wkt_loads('POLYGON ((-122.421611012467 37.80395991667648, -122.420227786592 37.80413210674329, -122.420159873957 37.80408277780808, -122.420117832801 37.80383748120349, -122.42154480663 37.80365748294528, -122.421611012467 37.80395991667648))'), {
+                               u'way_area': u'6936.45', u'source': u'openstreetmap.org', u'water': u'lagoon', u'name': u'Francisco Reservoir', u'covered': u'no'}))
+
+        self.assert_has_feature(
+            16, 10481, 25324, 'water',
+            {'kind': 'lake', 'alkaline': True, 'kind_detail': 'lake'})
```

Comment:
With the new `natural=water` tag addition above, I'd expect to see output more like:

>             {'kind': 'water', 'alkaline': True, 'kind_detail': 'lake'})